<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>942</width>
    <height>551</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RspMatch09-GUI_By_lcf1990@HUST</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextBrowser" name="TextOutPut">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>201</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>10</y>
      <width>241</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string>目标反应谱</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout">
       <property name="horizontalSpacing">
        <number>16</number>
       </property>
       <item row="1" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>T_g</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>alpha_max</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEdit_Tg"/>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEdit_alpha_max"/>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="pushButton_save_spec">
         <property name="text">
          <string>保存规范反应谱</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="pushButton_Calculate_sepc">
         <property name="text">
          <string>计算规范反应谱</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="pushButton_PlotRsp">
       <property name="text">
        <string>绘制规范反应谱</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>931</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <widget class="QCustomPlot" name="customplot_Rsp" native="true">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>20</y>
       <width>491</width>
       <height>141</height>
      </rect>
     </property>
    </widget>
    <widget class="QCustomPlot" name="customplot_Acc" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>381</width>
       <height>141</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>241</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>输入文件参数</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="3" column="1">
      <widget class="QComboBox" name="iModel"/>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="Gamma"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>MaxIter</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Tol</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>iModel</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Gamma</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="MaxIter"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="Tol"/>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="ShowSetting">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton_Show_AccDlg">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>190</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>plotAcc</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>190</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Table</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_DefineInpDlg_open">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Define inp File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_run_RspMatch">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>250</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>运行RspMatch</string>
    </property>
   </widget>
   <zorder>groupBox_3</zorder>
   <zorder>TextOutPut</zorder>
   <zorder>groupBox</zorder>
   <zorder>groupBox_2</zorder>
   <zorder>pushButton_Show_AccDlg</zorder>
   <zorder>pushButton</zorder>
   <zorder>pushButton_DefineInpDlg_open</zorder>
   <zorder>pushButton_run_RspMatch</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>942</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOpenFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpenFile"/>
   </widget>
   <addaction name="menuOpenFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpenFile">
   <property name="text">
    <string>OpenFile</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
